cfg.sample.toml

```diff
@@ -10,6 +10,23 @@ access_token = ""
 app_client_id = ""
 app_client_secret = ""
 
+# A base shared secret; this can be used to generate the per-repo github secret,
+# which is what's used to verify authenticity of github -> Homu webhooks. The
+# per-repo secret can be computed as follows (you'll need to enter this in
+# the repo webhook configuration):
+#
+# $ echo -n "$owner.$name" | openssl sha1 -hmac "$secret"
```

Comment:
As far as using SHA1...sure, though the github shared secret is a maximum of 20...characters?  UTF-8?  I don't know.  We could likely fit enough bits to do sha256 if we base64 encoded.